lib/js/app/queryCreator/components/Analysis/Analysis.tsx

```diff
@@ -93,27 +137,58 @@ const Analysis: FC<Props> = ({ analysis, onChange }) => {
         <AnalysisTitle>{transformName(analysis)}</AnalysisTitle>
       </DropableContainer>
       <Dropdown isOpen={isOpen}>
-        <Groups>
-          {ANALYSIS_GROUPS.map((options, idx) => (
-            <List key={idx}>
-              {options.map(({ label, value, index, description }) => (
-                <ListItem
-                  key={value}
-                  isActive={selectionIndex === index}
-                  description={description}
-                  analysis={value}
-                  onMouseEnter={() => setIndex(index)}
-                  onClick={(_e, analysis) => {
-                    setOpen(false);
-                    onChange(analysis);
-                  }}
-                >
-                  {label}
-                </ListItem>
-              ))}
-            </List>
-          ))}
-        </Groups>
+        <ScrollWrapper>
+          <Groups ref={dropdownRef}>
+            {ANALYSIS_GROUPS.map((options, idx) => (
+              <List key={idx}>
+                {options.map(({ label, value, index, description }) => (
+                  <ListItem
+                    key={value}
+                    isActive={selectionIndex === index}
+                    description={description}
+                    analysis={value}
+                    onMouseEnter={() => setIndex(index)}
+                    onClick={(_e, analysis) => {
+                      setOpen(false);
+                      onChange(analysis);
+                    }}
+                    showHint={(value, topPos, bottomPos) => {
+                      if (topPos && bottomPos && value) {
```

Comment:
This condition should be changed as `bottomPos` and `topPos` always exist. IMO the `setTooltip` should not be called for `topPos` or `bottomPos` === 0.  Please adjust this condition for better readability.